resource-management/cmds/service-api/app/server.go

```diff
@@ -45,7 +45,7 @@ type Config struct {
 func Run(c *Config) error {
 	klog.V(3).Infof("Starting the API server...")
 
-	store := redis.NewRedisClient()
+	store := redis.NewRedisClient(c.MasterIp, true)
```

Comment:
this will cause error when service started:
```
E0908 02:06:03.279489    3845 redis.go:62] Flush all dbs in error : (dial tcp 10.128.0.52:6379: connect: connection refused)
exit status 1
```

Reply:
Figure out the root cause is /etc/redis/redis.conf, need:
```
bind 0.0.0.0
protected-mode no
```

Reply:
The issue is because remote redis support configuration is not enabled in the file /etc/redis/redis.conf on the redis server.